codegen-core/common-test-models/simple.smithy

```diff
@@ -132,3 +132,22 @@ structure StoreServiceBlobInput {
 structure StoreServiceBlobOutput {
 
 }
+
+@http(method: "POST", uri: "/default-value")
```

Comment:
This should go into a Kotlin unit test for the BuilderGenerator so that it can be more thoroughly tested.